sdk/maps/Azure.Maps.Geolocation/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
```

Comment:
We'll want to keep the standard section header; there's automated tooling that keys on these.
```suggestion

### Features Added

```